plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergConnectorSmokeTest.java

```diff
@@ -69,6 +72,286 @@ public void tearDown()
         deleteRecursively(metastoreDir.toPath(), ALLOW_INSECURE);
     }
 
+    @Test
+    public void testLackOfPartitionFilterNotAllowed()
```

Comment:
I'd leave one typical test case here in the smoke test and move the rest to `BaseIcebergConnectorTest`

Reply:
Oh. Yes, smoke tests are designed to be quick and simple